src/app/app.module.js

```diff
@@ -7,20 +7,23 @@ angular.module('esn.inbox-calendar', [
   'esn.calendar.libs',
   'esn.resource.libs',
   'linagora.esn.videoconference',
-  'linagora.esn.videoconference.calendar'
+  'linagora.esn.videoconference.calendar',
+  'esn.lodash-wrapper'
 ]);
 require('esn-frontend-common-libs/src/frontend/js/modules/session.js');
 require('esn-frontend-common-libs/src/frontend/js/modules/notification.js');
 require('esn-frontend-common-libs/src/frontend/js/modules/i18n/i18n.module.js');
 
 require('esn-frontend-calendar/src/esn.calendar.libs/app/app.module.js');
 require('esn-frontend-calendar/src/esn.resource.libs/app/app.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/lodash-wrapper.js');
 
 require('./app.config');
 require('./app.constants');
 require('./invitation-message/blue-bar/invitation-message-blue-bar.component.js');
 require('./invitation-message/blue-bar/invitation-message-blue-bar.controller.js');
 require('./invitation-message/indicator/invitation-message-indicator.component.js');
+
```

Comment:
Why?

Reply:
@tuanlc Did you ask me to add a blank line `before`?

Reply:
Yes, but the blank line at the end of the file but not here